Translations/nl.js

```diff
@@ -0,0 +1,493 @@
+_global.translation['nl'] = {
+
+    //HTMLCSAuditor.js
+    "auditor_name" : 'HTML_CodeSniffer door Squiz'
+    ,"auditor_using_standard" : 'Gebruikt de standaard'
+    ,"auditor_standards" : 'Standaarden'
+    ,"auditor_code_snippet" : 'Codefragment'
+    ,"auditor_close" : 'Sluiten'
+    ,"auditor_select_types" : 'Kies welk type problemen op te nemen in het verslag'
+    ,"auditor_home" : 'Home'
+    ,"auditor_view_report" : 'Bekijk het verslag'
+    ,"auditor_report" : 'Verslag'
+    ,"auditor_back_to_report" : 'Terug naar het verslag'
+    ,"auditor_previous_issue" : 'Vorig probleem'
+    ,"auditor_next_issue" : 'Volgend probleem'
+    ,"auditor_issue" : 'Probleem'
+    ,"auditor_of" : 'van'
+    ,"auditor_errors" : 'Fouten'
+    ,"auditor_error" : 'Fout'
+    ,"auditor_warnings" : 'Waarschuwingen'
+    ,"auditor_warning" : 'Waarschuwing'
+    ,"auditor_notices" : 'Kennisgevingen'
+    ,"auditor_notice" : 'Kennisgeving'
+    ,"auditor_toggle_display_of" : 'Schakel weergave van'
+    ,"auditor_messages" : 'berichten'
+    ,"auditor_unable_to_point" : 'Kan niet wijzen naar het element waarop dit probleem betrekking heeft.'
+    ,"auditor_unable_to_point_entire" : 'Kan niet wijzen naar dit probleem, aangezien het betrekking heeft op het hele document'
+    ,"auditor_unable_to_point_removed" : 'Kan niet wijzen naar dit element, het werd verwijderd sedert de aanmaak van dit verslag'
```

Comment:
'sinds' is less archaic than 'sedert'. My personal choice would go to 'na' rather than 'sinds', but either choice is correct.